armi/reactor/blocks.py

```diff
@@ -1944,40 +1940,27 @@ def getPinCoordinates(self):
         Notes
         -----
         This assumes hexagonal pin lattice and needs to be upgraded once more generic geometry
-        options are needed.
-
-        A block with fully-defined pins could just use their individual spatialLocators in a
-        block-level 2-D grid. However most cases do not have this to minimize overhead and maximize
-        speed. Thus we want to just come up with a uniform mesh of pins if they're not explicitly
-        placed in the grid.
-
+        options are needed. Only works if pins have clad.
```

Comment:
I guess this should be clarified that it only works if there is a component with the `CLAD` flag, right? A user could potentially have a clad component with other flags, and this would fail.

Reply:
yeah, I'll clarify that in the docstring